hack/update-codegen.sh

```diff
@@ -43,11 +43,10 @@ setgen=$(kube::util::find-binary "set-gen")
 # update- and verify- scripts.
 ${clientgen} "$@"
 ${clientgen} -t "$@"
-
+${clientgen} --clientset-name="release_1_3" --input="api/v1,extensions/v1beta1"
```

Comment:
Actually just batch/v1 and autoscaling/v1, since only they are added in the internalclientset, I think that means people only need these two for now.